rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -2169,7 +2172,7 @@ message ClaimSwapNeuronsRequest {
 
 // An enum for representing the various statuses a Neuron being claimed by the
 // `claim_swap_neurons` API may have. The status is reported back to callers of
-// the API (mainly the SNS Sale canister) to indicate the success of the
+// the API (mainly the SNS decentralization swap canister) to indicate the success of the
```

Comment:
While the process is indeed referred to as "decentralization swap," the canister itself is called just (capitalized) Swap (or the SNS Swap canister).